README.md

```diff
@@ -1,5 +1,7 @@
 # Mixxx 
+[![GitHub release](https://img.shields.io/github/release/mixxxdj/mixxx.svg)](https://github.com/mixxxdj/mixxx/releases/latest)
```

Comment:
Please use https://img.shields.io/github/tag/mixxxdj/mixxx.svg = _GitHub tag (latest SemVer)_ for selecting the most recent version. Selecting releases by date is confusing and may alternate between different release branches. Currently it is showing 2.1.6.